README.md

```diff
@@ -101,6 +101,20 @@ switched on by default).
 
 <img src='./img/config.png' alt='Config' />
 
+
+### Description Generation
+Description generation works by taking the prompt prefix given in the **Config** sheet, appending a row of data from **Input** and sending the result as a prompt to LLM. This gives you great flexibility in shaping the wording, style and other requirements you might have. All data from **Input Feed** will be provided as part of the prompt.
+
+*Optional*: You can also provide examples of descriptions to items in **Few-shot** examples section (see below). Those will be appended to the prompt prefix as well and show to the model what *good* descriptions should look like.
```

Comment:
in "the" **Few-shot** examples section
show the model what good descriptions look like